Resources/public/js/models/ez-contentmodel.js

```diff
@@ -339,6 +339,24 @@ YUI.add('ez-contentmodel', function (Y) {
         },
 
         /**
+         * Creates a new draft for the content
+         *
+         * @method createDraft
+         * @param {Object} options
+         * @param {Object} options.api (required) the JS REST client instance
+         * @param {String} versionNo (optional) Version Number for the draft to be based on. If not
+         *                                      provided, the current version of the content will
+         *                                      be used. Ex: 42
+         * @param {Function} callback
+         */
+        createDraft: function (options, versionNo, callback) {
+            var capi = options.api,
+                contentService = capi.getContentService();
+
+            contentService.createContentDraft(this.get('id'), versionNo, callback);
```

Comment:
you should probably parse the response and provide a VersionInfoModel to the callback instead of the response Object